package.json

```diff
@@ -50,13 +50,9 @@
     "@babel/runtime": "^7.20.13",
     "@expo/html-elements": "^0.2.0",
     "@react-native-community/eslint-config": "^3.1.0",
-    "@storybook/addon-actions": "^7.0.0-rc.1",
-    "@storybook/addon-essentials": "^7.0.0-rc.1",
-    "@storybook/addon-links": "^7.0.0-rc.1",
-    "@storybook/builder-webpack5": "^7.0.0-rc.1",
-    "@storybook/manager-webpack5": "^7.0.0-alpha.10",
-    "@storybook/react": "^7.0.0-rc.1",
-    "@storybook/react-webpack5": "^7.0.0-rc.1",
+    "@storybook/addon-essentials": "^7.0.0-beta.0 || ^7.0.0-rc.0 || ^7.0.0",
```

Comment:
These are all dev dependencies so I don't think we need to have such a range of versions. Probably just ^7.0.0 is fine.